src/main.rs

```diff
@@ -262,20 +273,48 @@ fn exec(cmd: Command) -> Result<String> {
                 dest_wasm.display().to_string().bold()
             ))
         }
+        Command::Pack {
+            manifest_path,
+            verbosity,
+            unstable_options,
+        } => {
+            let manifest_path = ManifestPath::try_from(manifest_path.as_ref())?;
+            let metadata_result = cmd::metadata::execute(
+                &manifest_path,
+                verbosity.try_into()?,
+                false,
+                unstable_options.try_into()?,
+            )?;
+            let pack_result = cmd::metadata::execute(
+                &manifest_path,
+                verbosity.try_into()?,
+                true,
+                unstable_options.try_into()?,
+            )?;
+            Ok(format!(
+                "\nYour contract is ready. You can find it here:\n{}
+                \nYour metadata file is ready. You can find it here:\n{}
+                \nYour packed contract is ready. You can find it here:\n{}",
```

Comment:
```suggestion
                "\nYour contract's code is ready. You can find it here:\n{}
                \nYour contract's metadata is ready. You can find it here:\n{}
                \nYour contract bundle (code + metadata) is ready. You can find it here:\n{}",
```

Strictly speaking the wasm file is only the contract's code. A contract is the instantiation of that code. I am not suggesting that for splitting hairs. I think the message should make more clear what every artefact is.